app/connectors/SdesConnector.scala

```diff
@@ -75,18 +76,16 @@ class SdesConnector @Inject()(httpClient: HttpClient,
     )
   }
 
-  private def getSdesFiles[A, B <: SdesFile](url: String,
+  private def getSdesFiles[A, B <: SdesFile](urlLink: String,
                                              key: String,
                                              metricsName: String,
                                              transform: Seq[A] => Seq[B])
-                                            (implicit reads: HttpReads[HttpResponse],
-                                             readSeq: HttpReads[Seq[A]]): Future[Seq[B]] = {
+                                            (implicit readSeq: HttpReads[Seq[A]]): Future[Seq[B]] = {
     metricsReporterService.withResponseTimeLogging(metricsName) {
-      httpClient.GET[HttpResponse](
-        url,
-        headers = Seq("x-client-id" -> appConfig.xClientIdHeader, "X-SDES-Key" -> key)
-      )(reads, HeaderCarrier(), implicitly)
-        .map(readSeq.read("GET", url, _))
+      httpClient.get(url"$urlLink")(HeaderCarrier())
+        .setHeader(("x-client-id" -> appConfig.xClientIdHeader), ("X-SDES-Key" -> key))
```

Comment:
Could we move the Header keys in a separate object please? We do this in other frontend repos.